ddtrace/runtime_metrics/collector.py

```diff
@@ -0,0 +1,87 @@
+import importlib
+import logging
+
+log = logging.getLogger(__name__)
```

Comment:
```python
from ..internal.logger import get_logger

log = get_logger(__name__)
```

This is the new rate limited logger, so you can do `log.warn`, `log.error`, and `log.debug` (don't use any other levels) without any issues.